sdk/servicebus/service-bus/src/constructorHelpers.ts

```diff
@@ -47,10 +47,19 @@ export function createConnectionContextForConnectionString(
   config.webSocketConstructorOptions = options?.webSocketOptions?.webSocketConstructorOptions;
 
   const credential = new SharedKeyCredential(config.sharedAccessKeyName, config.sharedAccessKey);
-  ConnectionConfig.validate(config);
+  validate(config);
   return ConnectionContext.create(config, credential, options);
 }
 
+function validate(config: ConnectionConfig) {
+  // TODO: workaround - core-amqp's validate string-izes "undefined"
```

Comment:
I'd have expected us to make the change in core-amqp so that it sets the entity path to be an empty string in such cases in the same PR. Any reason we chose to do that at a later time?

Reply:
I was unclear about this one because the fix is straightforward but I wasn't sure if anybody was "depending" on it working that way. So I just need to go through EH. Can file an issue.

Reply:
Filed issue: #8105